suites/nautilus/ansible/tier-2_deploy_test-ceph-rpms.yaml

```diff
@@ -234,3 +234,43 @@ tests:
       polarion-id: CEPH-83573277
       module: purge_dashboard.py
       desc: Purges ceph dashboard
+
+  - test:
+      name: Ceph ansible log validation
+      polarion-id: CEPH-83572701
+      module: test_ansible.py
+      config:
```

Comment:
@adityaramteke why you need all these config here?

Reply:
@sunilangadi2 sure, we can do that!

Reply:
@sunilangadi2 the requested changes are updated! :-)

Reply:
Thanks @adityaramteke changes LGTM.